frontend/src/types.ts

```diff
@@ -616,8 +616,10 @@ export interface FilterType {
     filter_test_accounts?: boolean
     from_dashboard?: boolean
     funnel_step?: number
-    funnel_viz_type?: string // this and the below param is used for funnels time to convert, it'll be updated soon
+    funnel_viz_type?: string // this and funnel_to_step is used for funnels time to convert
     funnel_to_step?: number
+    entrance_period_start?: string // this and drop_off is used for funnels time conversion date for the persons modal
```

Comment:
@alexkim205 I've added a comment here for it, originally I was going to rename it but there's a pretty detailed explanation of this in the backend and renaming it might make the explanation there confusing

Reply:
👍